### ISSUE TITLE
Wrong vocabulary overview diagram

### ISSUE THREAD

**Issue by jakubklimek** [opendata-mvcr/termit-ui#56]
In [this vocabulary](https://kbss.felk.cvut.cz/termit-kodi/#/vocabularies/2013?namespace=https://slovník.gov.cz/legislativní/sbírka/304/), I have 4 terms - 1 event, 1 object type, 2 objects. However, the diagram shows 75% undefined 25% object.
![image](https://user-images.githubusercontent.com/718827/100766359-d9910d00-33f8-11eb-946a-75a1c14679d9.png)

**Comment by ledsoft** [comment]
The statistics query used by TermIt apparently takes into account only the following types:
* typ-objektu
* typ-vlastnosti
* typ-vztahu
* typ-události

So the event and two objects you mention will be treated as un-typed.

@psiotwo care to shed some light on this restriction? (You created the original statistics query, I just adapted it for the workspace-based scenario).

**Comment by psiotwo** [comment]
Yes, this is a bug. Will resolve it as a part of https://kbss.felk.cvut.cz/redmine/issues/1460 first.
